client/finality-grandpa/src/communication/mod.rs

```diff
@@ -474,12 +474,12 @@ fn incoming_global<B: BlockT>(
 		gossip_validator: &Arc<GossipValidator<B>>,
 		voters: &VoterSet<AuthorityId>,
 	| {
-		let precommits_signed_by: Vec<String> =
-			msg.message.auth_data.iter().map(move |(_, a)| {
-				format!("{}", a)
-			}).collect();
-
 		if voters.len() <= TELEMETRY_VOTERS_LIMIT {
+			let precommits_signed_by: Vec<String> =
```

Comment:
I'm dumb, I added this guard since you had previously said there were a lot of allocations in gossip telemetry, and apparently I forgot to move this inside the guard.